### ISSUE TITLE
PyPI release

### ISSUE THREAD

**Issue by Fusselwurm** [LordGolias/sqf#62]
can we get releases on PyPI again ? I really like to be able to just use `pip` to get an up-to-date sqflint CLI – and the latest version 0.3.2 does not know recent additions like `findIf`

**Comment by vincentBenet** [comment]
yes, i have to use it as submodule but imports are messy... (made a PR)
